woodwork/type_sys/utils.py

```diff
@@ -40,6 +41,8 @@ def _is_numeric_series(series, logical_type):
     for the purposes of determining if it can be a time_index."""
     if ks and isinstance(series, ks.Series):
         series = series.to_pandas()
+    if dd and isinstance(series, dd.Series):
+        series = series.compute()
```

Comment:
Can we use only the first partition here as well?

Reply:
yep!

Reply:
I don't think I meant to add it to `_is_valid_latlong_series` in the first place, but I guess it's a part of type inference, so maybe we actually want it there...